docs/pages/web/toast.js

```diff
@@ -45,30 +28,223 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
             type="do"
             title="When to use"
             description={`
-          - Displaying non-critical feedback on the result of an action.
-          - Reinforcing success at the surface level.
+- Briefly acknowledging a user action without interrupting their flow
+- When acknowledging an action that relates to another surface, providing a link that navigates the user to that surface
+- To undo actions after acknowledgement, if there isn’t already a way to do so on the current surface
+- For system processes like showing that a process is loading, or when there are internet connectivity issues
         `}
           />
           <MainSection.Card
             cardSize="md"
             type="don't"
             title="When not to use"
             description={`
-          - Providing an update related to anything other than confirmation of a successful action. Consider a [Callout](/web/callout) instead.
-          - Presenting mandatory and/or critical actions to a user.
-          - Displaying feedback at the element level (e.g., entered password doesn't meet requirements). Use inline text instead.
+- When, due to an error, a user can’t even continue performing basic tasks like browsing already loaded Pins.
+- When asking a user to confirm that they want to perform an action. Use [AlertModal](/web/alertmodal) instead
+- When you want to suggest a user spend more money or try new features; use [Upsell](/web/upsell) instead.
+- For errors that relate to a specific section or page. Use Callout(/web/callout) or SlimBanner(/web/slimbanner) instead.
+- To guide or educate the user. Use Popover(/web/popover) or Tooltip(/web/tooltip) instead.
         `}
           />
         </MainSection.Subsection>
       </MainSection>
 
-      <AccessibilitySection name={generatedDocGen?.displayName} />
+      <AccessibilitySection name={generatedDocGen?.displayName}>
+        <MainSection.Subsection
+          title="Icons"
+          description={`
+\`iconAccessibilityLabel\` requires a short, descriptive label for screen readers. This label should communicate the intent of the icon, such as "Success", “Error”, “Link”. They should also be localized.
```

Comment:
grammar nit: "they" doesn't agree with "iconAccessibilityLabel". It's a single label, so "it" would be more appropriate.